mrtt-ui/src/components/RestorationAimsForm/RestorationAimsForm.js

```diff
@@ -43,24 +42,12 @@ const RestorationAimsForm = () => {
     reset: resetForm
   } = reactHookFormInstance
 
-  const _loadSiteData = useEffect(() => {
-    if (apiAnswersUrl && resetForm) {
-      setIsLoading(true)
-      axios
-        .get(apiAnswersUrl)
-        .then(({ data }) => {
-          setIsLoading(false)
-          const initialValuesForForm = formatApiAnswersForForm({
-            apiAnswers: data,
-            questionMapping: questionMapping.restorationAims
-          })
-          resetForm(initialValuesForForm)
-        })
-        .catch(() => {
-          toast.error(language.error.apiLoad)
-        })
-    }
-  }, [apiAnswersUrl, resetForm])
+  usePopulateQuestionFormWithInitialValues({
```

Comment:
i'm confused by the naming - we are just doing a fetch to get initial values right? and using the question mapping?

Reply:
ya, its just for forms that use the question mapping. Dont love that the new site form doesnt. Its got a different payload being sent than the rest


`useInitializeQuestionMappedForms`?

Reply:
what about `initializeQuestionFormData` and `initializeSiteFormData`?